.circleci/config.yml

```diff
@@ -1,7 +1,79 @@
 version: 2.1
 orbs:
-  node: circleci/node@3.0.0
+  node: circleci/node@1.1.6
+jobs:
+  prod-build:
+    docker:
+      - image: uwbpparamedics/uw-blueprint-paramedics-react:0.0.1
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: npm install
```

Comment:
pretty sure this doesn't really benefit us as `docker-compose up` will implicitly install packages further down
```suggestion
```